### PR TITLE
Separate storybook from the root repo

### PR DESCRIPTION
**Pull request by SemraSh**
---
name: Separating storybook from the root repo
about: Fixing local linking for development
---

## Summary

Moving storybook to a separate repo.

## Motivation

Previously storybook held place inside the root repo. This generated problems with having React and React Dom as dev-dependencies. For local development linking 2 projects created multiple Reacts to be picked as the root directory required  React for storybook. In order to solve this issue I've created an additional repo for storybook which is not in packages, therefore, the node_modules of storybook is separated from the rest of the monorepo.

## Describe alternatives you've considered

The alternative would have been to remove the yarn-workspaces, however, this is not going to be a sustainable solution for the long term usage of the library. We want to keep the advantages of yarn-workspaces.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,47 +51,87 @@ This project is licensed under the terms of the
...
+cd storybook
+```
+
+install packages
 
+```bash
+yarn
 ```
+
+start srotybook
```
Comment (digigarlab) [inline README.md:76]: *storybook

Comment (digigarlab) [reply README.md:76]: What is this start command? Isn't it `yarn storybook`?

**Review by digigarlab — COMMENTED** [review]
Some of the linting is failing since the test packages were moved to devDependencies. This could be addressed by tweaking the lint rule as follow (untested):
```json
"import/no-extraneous-dependencies": ["error", {"devDependencies": ["**/*.spec.js"]}]
```

File: README.md (hunk 1 above)
Comment (SemraSh) [reply README.md:76]: yes indeed, that is me thinking something else while typing

Comment (SemraSh) [reply README.md:76]: it is `start-storybook -p 9000`

**Review by digigarlab — APPROVED** [review]

**Pull request merged**